src/NServiceBus.Metrics.ServiceControl/ServiceControlReporting/EndpointMetadata.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace NServiceBus.Metrics.ServiceControl.ServiceControlReporting
+{
+    class EndpointMetadata
+    {
+        public EndpointMetadata(string localAddress)
+        {
+            this.localAddress = localAddress;
+        }
+
+        public string ToJson()
+        {
+            return SimpleJson.SerializeObject(new
+            {
+                Version = 2,
```

Comment:
Why is this `2`? No comment about the version value.

Reply:
Name changed to `PluginVersion`